tests/integration_tests/README.md

```diff
@@ -76,7 +78,7 @@ We recommend that you provide docker with at least 6+ cores and 8G+ memory. Of c
 > We will try to resolve these issues as soon as possible.
 
 1. If you want to run kafka tests,
-   run `CASE="clustered_index" docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up --build`
+   run `START_AT="clustered_index" docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up --build`
```

Comment:
Why only change `run kafka test` to `START_AT`?

Reply:
to show the usage of `CASE` and `START_AT`t